src/data_utils.jl

```diff
@@ -217,17 +257,30 @@ table(::LightInterface, X; kw...) = errlight("table")
 """
     nrows(X)
 
-Return the number of rows for a table, abstract vector or matrix `X`.
+return the number of rows for a table, abstractvector or abtractmatrix, `X`.
```

Comment:
I would either revert to the original or use "`AbstractVector`", "`AbstractMatrix`".

Reply:
And `return` needs capitalisation.